ocaml/xapi/xapi_vm_migrate.ml

```diff
@@ -438,7 +438,9 @@ let pool_migrate ~__context ~vm ~host ~options =
         detach_local_network_for_vm ~__context ~vm ~destination:host ;
         Helpers.call_api_functions ~__context (fun rpc session_id ->
             XenAPI.VM.pool_migrate_complete ~rpc ~session_id ~vm ~host
-        )
+        ) ;
+        Xapi_vm_lifecycle.remove_pending_guidance ~__context ~self:vm
```

Comment:
If you do this in `pool_migrate_complete`, then it will work for both `VM.pool_migrate` as well as `VM.migrate_send`.

Reply:
Sorry, I missed the "storage live migration" case, thank you!